packages/integration-tests/cmr.js

```diff
@@ -105,8 +106,8 @@ async function waitForConceptExistsOutcome(cmrLink, expectation, retries = 3, in
         timeout: interval * retries
       }
     );
-
-    console.log(`waitForConceptExistsOutcome() succeeded. conceptExists('${cmrLink}') === ${expectation}`);
+    const shardSettleTimeMs = 1000;
+    await sleep(shardSettleTimeMs);
```

Comment:
What's the benefit of using `sleep` vs. simply increasing the interval/retries?

Reply:
The sleep happens after the pWaitFor has found the record.  This is because there are two shards that can be searched and it can take up to a second for them to reconcile (or whatever the DB term is?) so we were seeing that after waiting for the condition to be true, if we exit this function and immediately call conceptExists again, we can get a different response.

From our debugging with Chris Durbin
```
The theory:
1.) Every document we write gets written to a primary and a replica shard.
2.) The refresh interval is set to 1s (a document will be visible for search after 1 second).
3.) The refresh interval is independent for the primary and replica and so the refresh will be performed at different times.
4.) Until the refresh has occurred after indexing a document, it will be unavailable in search results.
5.) A search will hit be executed against one of the two shards (might just round robin, not sure).
6.) So search 271 searched the primary shard and 272 searched the replica shard (or vice versa - it's just important that the search hit different shards)
7.) When 271 was executed (although it had been less than a second since the indexing occurred for the document) the refresh interval had occurred and so the document was visible.
8.) When 272 was executed the refresh interval had still not expired and so the document was not visible on that shard.
... so my suggestion is to add a 2 second sleep after waitForConceptExistsOutcome.
```
(I've run 3400 tests against a 1 seconds timeout and haven't seen any failures.)

Reply:
So essentially this isn't a guard against the call to `conceptExists` inside the `pWaitFor` but for the subsequent call to `conceptExists` afterward?

Reply:
I see now, thanks @flamingbear, that makes a lot more sense.

Reply:
@markdboyd exactly.